plugins/__init__.py

```diff
@@ -0,0 +1,4 @@
+import os
+import sys
+
+sys.path.append(os.path.dirname(__file__))
```

Comment:
So adding this code to this __init__.py allowed GH actions to successfully find the modules used in the test